pkg/ip/cidr.go

```diff
@@ -30,3 +31,38 @@ func ParseCIDRs(cidrs []string) (valid []*net.IPNet, invalid []string) {
 	}
 	return valid, invalid
 }
+
+// PrefixToIPNet is a convenience helper for migrating from the older 'net'
+// standard library types to the newer 'netip' types. Use this to plug the
+// new types in newer code into older types in older code during the migration.
+func PrefixToIPNet(prefix netip.Prefix) *net.IPNet {
+	return &net.IPNet{
+		IP:   prefix.Addr().AsSlice(),
+		Mask: net.CIDRMask(prefix.Bits(), prefix.Addr().BitLen()),
+	}
+}
+
+// IPNetToPrefix is a convenience helper for migrating from the older 'net'
+// standard library types to the newer 'netip' types. Use this to plug the
+// new types in newer code into older types in older code during the migration.
+func IPNetToPrefix(prefix *net.IPNet) netip.Prefix {
+	if prefix == nil {
+		return netip.Prefix{}
+	}
+	ip, _ := netip.AddrFromSlice(prefix.IP)
+	ones, _ := prefix.Mask.Size()
```

Comment:
Should we validate the number of bits here and return an empty `netip.Prefix` in case it's not a valid IPv4 or IPv6 mask, i.e.

```suggestion
	ones, bits := prefix.Mask.Size()
	if bits != net.IPv4len*8 && bits != net.IPv6len*8 {
		// invalid mask
		return netip.Prefix{}
	}
```